recipe/meta.yaml

```diff
@@ -15,13 +15,13 @@ build:
 
 requirements:
   host:
-    - python >=3.10
+    - python >=3.11,<3.14
```

Comment:
Here, could you please follow the example here? We need to use `python {{ python_min }}` variables:

https://github.com/conda-forge/diffpy.fourigui-feedstock/issues/1

In this PR, we can also close this PR

Reply:
done